policies.go

```diff
@@ -589,19 +627,47 @@ func (t *tokenAwareHostPolicy) Pick(qry ExecutableQuery) NextHost {
 	}
 
 	token := partitioner.Hash(routingKey)
-	ht := meta.replicas[qry.Keyspace()].replicasFor(token)
 
 	var replicas []*HostInfo
-	if ht == nil {
-		host, _ := meta.tokenRing.GetHostForToken(token)
-		replicas = []*HostInfo{host}
+
+	t.tablets.mu.Lock()
+	tablets := t.tablets.get()
+
+		// Search for tablets with Keyspace and Table from the Query
+		l := findTablets(tablets, qry.Keyspace(), qry.Table())
```

Comment:
Nit: wrong indentation.

Reply:
Are you sure? I looks fine to me. This lines are inside of "if" and it is inside of function so we have two tabs

Reply:
When you look solely at ddd0d7f46431fb5a1d62de75f17297a5686bed16 the indentation is incorrect, but 3b66037f379ae4d98793e0299086546a5853f462 "fixes" it (by adding the outer ifs).

Reply:
Fixed